demosplan/DemosPlanCoreBundle/Command/Addon/AddonUninstallCommand.php

```diff
@@ -120,6 +138,27 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         return self::SUCCESS;
     }
 
+    private function uninstallAddon(AddonInfo $addonInfo, SymfonyStyle $output): void
```

Comment:
u can use this method to replace the duplicated code from line 111

Reply:
done in https://github.com/demos-europe/demosplan-core/pull/3548/commits/4f85edc437768215af43d942b7fd47fd35d156ac